src/view/configuration/dataCollectionSection.jsx

```diff
@@ -207,6 +213,19 @@ const DataCollectionSection = ({ instanceFieldName }) => {
                   defaultValue={instanceDefaults.downloadLinkQualifier}
                 />
               </Flex>
+              <Flex gap="size-100">
+                <CodeField
+                  data-test-id="onBeforeLinkClickSendEditButton"
+                  label="On before link click send callback"
+                  buttonLabelSuffix="on before link click event send callback code"
+                  name={`${instanceFieldName}.onBeforeLinkClickSend`}
+                  description='Callback function for modifying data before each link click event is sent to the server. A variable named "content" will be available for use within your custom code. Filter by "content.clickedElement" or modify "content.xdm" or "content.data" as needed to transform data before it is sent to the server. Return "false" if you want to cancel the link tracking for this element.'
```

Comment:
I would remove the quotes around false. They may try to return the string "false".